tutorials/plot_eeg_no_mri.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+"""
+EEG forward operator with a template MRI
+========================================
+
+This tutorial explains how to compute the forward operator from EEG data
+using the standard template MRI subject ``fsaverage``.
+
+.. important:: Source reconstruction without an individual T1 MRI from the
+               subject will be less accurate. Do not over interpret
+               activity locations which can be off by multiple centimeters.
+
+.. contents:: This tutorial covers:
+   :local:
+   :depth: 2
+
+"""
+
+# Authors: Alexandre Gramfort <alexandre.gramfort@inria.fr>
+#          Joan Massich <mailsik@gmail.com>
+#
+# License: BSD Style.
+
+import os.path as op
+
+import mne
+from mne.datasets import set_montage_coreg_path, fetch_fsaverage, sample
+
+# Convenience function to set ``subjects_dir`` default value for users who
+# only ever plan to do montage-based coreg with fsaverage.
+subjects_dir = set_montage_coreg_path()
```

Comment:
For users with `SUBJECTS_DIR` set, this does nothing. For EEG users without such a value, `SUBJECTS_DIR` in MNE config gets set to `~/mne_data/MNE-fsaverage-data`.